compiler/rustc_ast_lowering/src/item.rs

```diff
@@ -723,7 +723,10 @@ impl<'hir> LoweringContext<'_, 'hir> {
         }
     }
 
-    fn lower_field_def(&mut self, (index, f): (usize, &FieldDef)) -> hir::FieldDef<'hir> {
+    pub(super) fn lower_field_def(
```

Comment:
Stray change?

Reply:
If use the function somewhere else outside of this module.